content/Video_Display.md

```diff
@@ -670,10 +670,11 @@ sprite in OAM (\$FE00-\$FE03) has the highest priority, and so on. In
 Non-CGB mode, the smaller the X coordinate, the higher the priority. The
 tie breaker (same X coordinates) is the same priority as in CGB mode.
 
-The priority calculation between sprites disregards OBJ-to-BG Priority
-(attribute bit 7). Only the highest-priority nonzero sprite pixel at any
-given point is compared against the background. Thus if a sprite with a
-higher priority (based on OAM index) but with OBJ-to-BG Priority turned
+Priority among pixels that overlap is determined using the rules explained
+above. After the pixel with the highest priority has been determined,
+the OBJ-to-BG priority of only that pixel is honored (or disregarded if
+this is a zero pixel). Thus if a sprite with a
```

Comment:
I see what you mean, but I think it may help to mention that these are transparent pixels. Pointing out that these are pixels with index 0 (hence, regardless of palette) is probably a good idea indeed.